tests/s3_api_gradle.erl

```diff
@@ -1,33 +1,54 @@
 -module(s3_api_gradle).
--include_lib("eunit/include/eunit.hrl").
 
 -behavior(riak_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
 -export([
     confirm/0
 ]).
 
 -define(NODE_COUNT, 3).
+-define(TEST_USER, ["test", "test@example.com"]).
+-define(METADATA_KEY, [{<<"security">>, <<"users">>}, <<"test">>]).
 
 confirm() ->
-    Nodes = rt:deploy_nodes(?NODE_COUNT),
-    ok = rt:wait_until_nodes_ready(Nodes),
+    %% Build a simple cluster
+    Nodes = rt:build_cluster(?NODE_COUNT),
+
+    %% Choose random node to interact with
+    Node = lists:nth(random:uniform(?NODE_COUNT), Nodes),
 
-    %% Set to testing profile backend on all the nodes
-    {[ok, ok, ok], []} = rpc:multicall(Nodes,
-                                       application,
-                                       set_env,
-                                       [riak_s3_api, default_user_backend, riak_s3_user_profile_backend]),
+    %% Create test user and wait for it to appear on the cluster
+    _ = rpc:call(Node, riak_s3_user, create, ?TEST_USER),
+    rt:wait_until(fun() ->
+                        {Results, []} = rpc:multicall(Nodes, riak_core_metadata, get, ?METADATA_KEY),
+                        case lists:filter(fun(R) -> R == undefined end, Results) of
+                            [] -> false;
+                            _ -> true
+                        end
+                  end),
+    [
+        {"s3.access_key_id", S3AccessKey},
+        {"s3.display_name", _},
+        {"s3.secret_access_key", S3SecretKey}
+    ] = rpc:call(Node, riak_core_metadata, get, ?METADATA_KEY),
 
```

Comment:
Are we guaranteed to get the 3 parameters (s3.access_key_id/display_name/secret_assecc_key) in the same order every time, on every platform? Otherwise, I'd sort the result and the list your pattern-matching on just to be sure.